jekyll/_cci2/sample-config.md

```diff
@@ -27,8 +27,10 @@ version: 2
 jobs:
   build:
     working_directory: ~/mern-starter
+    # The first image in this list runs on the primary container where all steps are run.
```

Comment:
Suggest: "The primary container is an instance of the first list image listed. Your build commands run in this container."